src/app/services/weather-api.service.spec.ts

```diff
@@ -1,16 +1,40 @@
 import { TestBed } from '@angular/core/testing'
+import { Data } from 'src/interfaces'
```

Comment:
weird, maybe you didn't commit this file, I don't see interfaces folder in github

Reply:
https://github.com/Alkowskey/BaseAngular/blob/test-countByDirection/src/interfaces.ts

it is there from the moment i did http requests and forms